src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -17199,11 +17199,11 @@ public bool MoveNext()
                         goto HeaderGrpcMessage;
                     case 2:
                         goto HeaderGrpcStatus;
-
+                    
                     default:
```

Comment:
too many trailing spaces

Reply:
I cleaned them up when I edited the file manually but the tests didn't like that. If you care, file an issue to clean up the trailing whitespace `KnownHeaders` generates.